analyzer-java/pom.xml

```diff
@@ -39,6 +46,11 @@
       <version>5.10.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>net.jimblackler.jsonschemafriend</groupId>
+      <artifactId>core</artifactId>
+      <version>0.12.2</version>
```

Comment:
If you disagree with the other comment and think the JSON schema check should only be in test code, this should get a

```
<scope>test</scope>
```

so that we don't ship this thing when it's not needed.